### PR TITLE
Bugfix infinite loop on 401 errors

### PR DESCRIPTION
**Pull request by maxmarkus**
Redirecting to request-error component on 401 if the token is younger then 10 seconds, else go to overview.

Fixes kyma/#635

### CODE REVIEW

File: core/src/app/auth/token.interceptor.ts [hunk 1]
```diff
@@ -9,10 +9,25 @@ import { OAuthService } from 'angular-oauth2-oidc';
 import { AppConfig } from './../app.config';
 import { Observable } from 'rxjs';
 import { tap } from 'rxjs/operators';
+import { Router } from '@angular/router';
+
+interface ITokens {
+  id_token_stored_at?: string;
+  id_token_expired_at?: string;
```
Comment (jesusreal) [inline core/src/app/auth/token.interceptor.ts]: It look like we are not using this variable :)

Comment (maxmarkus) [reply core/src/app/auth/token.interceptor.ts]: We're not using this whole interface anymore ^^.. thanks

File: core/src/app/auth/token.interceptor.spec.ts [hunk 2]
```diff
@@ -21,11 +50,20 @@ describe('TokenInterceptor', () => {
   let http: HttpClient;
 
   beforeEach(() => {
+    spyOn(sessionStorage, 'getItem').and.callFake(mockSessionStorage.getItem);
+    spyOn(sessionStorage, 'setItem').and.callFake(mockSessionStorage.setItem);
+    spyOn(sessionStorage, 'removeItem').and.callFake(
```
Comment (dariadomagala) [inline core/src/app/auth/token.interceptor.spec.ts]: Why do we have to set storage item and then remove it before each test?

Comment (maxmarkus) [reply core/src/app/auth/token.interceptor.spec.ts]: This is setting up spies and exchanges the dom implementation with custom one. Enables us to watch on getItem calls while still having the logic working.

**Pull request merged**